Packs/Zendesk/Integrations/Zendeskv2/README.md

```diff
@@ -43,7 +46,7 @@ Gets the specified user's data.
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
 | user_id | The user's ID to retrieve. | Optional | 
-| user_name | The user's name. | Optional | 
+| user_name | The user's name. <br/>Required permissions to use this argument: Agents . | Optional | 
```

Comment:
Why do you add the `<br/>` ?  It's an HTML tag

Reply:
Its auto-generated by the SDK generate-docs i guess it identifies the new breakline from the yml and pasted it here.

Reply:
![image](https://user-images.githubusercontent.com/95565843/217271420-382b788e-d00d-444f-9a32-985169a28d89.png)